tests/ballerina-test/src/test/java/org/ballerinalang/test/services/nativeimpl/request/RequestNativeFunctionNegativeTest.java

```diff
@@ -201,8 +201,8 @@ public void testCompilationErrorTestCases() {
         Assert.assertEquals(resultNegative.getErrorCount(), 2);
         //testRequestSetStatusCode
         BAssertUtil.validateError(resultNegative, 0,
-                "undefined function 'setStatusCode' in struct 'ballerina/http:Request'", 4, 5);
+                "undefined function 'setStatusCode' in struct 'ballerina.http:Request'", 4, 5);
         BAssertUtil.validateError(resultNegative, 1,
-                "undefined field 'statusCode' in struct 'ballerina/http:Request'", 5, 5);
+                "undefined field 'statusCode' in struct 'ballerina.http:Request'", 5, 5);
```

Comment:
Why has this been changed? Also wherever you find the name struct, change it to record.